packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -219,13 +219,15 @@ abstract class ResidentRunner {
 
     // Refresh the view list.
     await vmService.vm.refreshViews();
-    for (int i = 0; vmService.vm.mainView == null && i < 5; i++) {
+    for (int i = 0; vmService.vm.firstView == null && i < 5; i++) {
```

Comment:
Ditto about the parens.

This looks like it is duplicated code from above? Please refactor this into a helper function.